parcel.go

```diff
@@ -14,47 +16,143 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
+	res, err := s.db.Exec("INSERT INTO parcel (Client, Status, Address, Created_at) VALUES (:Client, :Status, :Address, :Created_at)",
+		sql.Named("Client", p.Client),
+		sql.Named("Status", p.Status),
+		sql.Named("Address", p.Address),
+		sql.Named("Created_at", p.Created_at))
+	if err != nil {
+		fmt.Println(err)
+		return 0, err
+	}
+
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, nil
+	}
 	// верните идентификатор последней добавленной записи
-	return 0, nil
+	return int(id), nil
+
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
 	// реализуйте чтение строки по заданному number
 	// здесь из таблицы должна вернуться только одна строка
 
 	// заполните объект Parcel данными из таблицы
-	p := Parcel{}
+	rows, err := s.db.Query("SELECT Number, Client, Status, Address, Created_at FROM parcel WHERE number = :number",
```

Comment:
Для выбора одной строки нужно использовать `s.db.QueryRow`, а потом  `row.Scan`
```suggestion
	rows, err := s.db.QueryRow("SELECT Number, Client, Status, Address, Created_at FROM parcel WHERE number = :number",
```